browse/config.py

```diff
@@ -255,3 +255,27 @@
 
 # arXiv Labs options
 LABS_BIBEXPLORER_ENABLED = os.environ.get('LABS_BIBEXPLORER_ENABLED', True)
+
+# Auth settings
+AUTH_SESSION_COOKIE_NAME = 'ARXIVNG_SESSION_ID'
+AUTH_SESSION_COOKIE_DOMAIN = os.environ.get(
+    'AUTH_SESSION_COOKIE_DOMAIN', '.arxiv.org')
+AUTH_SESSION_COOKIE_SECURE = bool(
+    int(os.environ.get('AUTH_SESSION_COOKIE_SECURE', '1')))
+AUTH_UPDATED_SESSION_REF = True
+
+CLASSIC_COOKIE_NAME = os.environ.get('CLASSIC_COOKIE_NAME', 'tapir_session')
+CLASSIC_PERMANENT_COOKIE_NAME = os.environ.get(
+    'CLASSIC_PERMANENT_COOKIE_NAME',
+    'tapir_permanent'
+)
+CLASSIC_TRACKING_COOKIE = os.environ.get('CLASSIC_TRACKING_COOKIE', 'browser')
+CLASSIC_DATABASE_URI = os.environ.get('CLASSIC_DATABASE_URI', os.environ.get(
+    'BROWSE_SQLALCHEMY_DATABASE_URI', 'sqlite:///../tests/data/browse.db'))
```

Comment:
This looks like configuration related to testing. It seems like that should be setup in the tests, not in the config?
Failure to correctly configure the env vars would result in a production system running a test db.

Reply:
This is configuration related to auth. Some options may not be needed for browse purposes. I'll review.

Reply:
I my concern here is that a missing configuration will spin up this code with a test DB. I'd prefer that missing configuration die with "missing CLASSIC_DB_URI" That might not be the model we are using though.

Reply:
What do you think about this tweak:

`CLASSIC_DATABASE_URI = os.environ.get('CLASSIC_DATABASE_URI', os.environ.get(
    'BROWSE_SQLALCHEMY_DATABASE_URI', None))`

Generally this should never hit the `None` case, but if it does, we could raise an error that prevents the app from starting. In the case where it's fallen back to sqlite, we could issue a warning.

Reply:
:+1: